shell.nix

```diff
@@ -1,26 +1,18 @@
-{ pkgs ? import (fetchGit {
-    url = https://github.com/dapphub/dapptools.git;
-    rev = "6943c76bfb8e0b1fce54c3d9bba6f0f7e50d2f5c";
-  }) {}
-, mcd-cli ? pkgs.callPackage (import (fetchGit {
-    url = https://github.com/makerdao/mcd-cli.git;
-    rev = "86842b49defa53301ac0019f7d5994859bb3e1e9";
-  })) {}
-, sethret ? (import (fetchGit {
-    url = https://github.com/icetan/sethret.git;
-    rev = "ef77915e2881011603491275f36b44bf2478b408";
-  }) {}).sethret
-}:
+{ pkgsSrc ? (import ./nix/pkgs.nix {}).pkgsSrc
```

Comment:
What's `pkgsSrc` and where is it defined?

Reply:
It comes from the pinning repo https://github.com/makerdao/nixpkgs-pin/blob/master/default.nix#L19